gesture_control:
  camera_index: 0
  frame_rate: 0.1

distance_sensor:
  serial_port: "/dev/ttyACM0"
  baud_rate: 115200

car_control:
  serial_port: "/dev/ttyACM0"
  baud_rate: 115200
  stop_distance: 20
